tests/e2e/main.go

```diff
@@ -254,29 +253,17 @@ var rootCmd = &cobra.Command{
 			}
 		}
 
-		// generate mao config
-		maoConfigTemplateData, err := ioutil.ReadFile(filepath.Join(assetsPath, "manifests/mao-config.yaml"))
+		installConfigData, err := ioutil.ReadFile(filepath.Join(assetsPath, "manifests/install-config.yaml"))
```

Comment:
Use https://golang.org/pkg/path/#Join for all elements. Here's there's an embedded `/`.